fabtests/scripts/runfabtests.sh

```diff
@@ -130,12 +130,10 @@ functional_tests=(
 	"fi_recv_cancel -e rdm -V"
 	"fi_unexpected_msg -e msg -I 10"
 	"fi_unexpected_msg -e rdm -I 10"
-	"fi_inj_complete -e msg"
-	"fi_inj_complete -e rdm"
-	"fi_inj_complete -e dgram"
-	"fi_inj_complete -e msg -SR"
-	"fi_inj_complete -e rdm -SR"
-	"fi_inj_complete -e dgram -SR"
+	"fi_msg_inject -A inject -v"
+	"fi_msg_inject -NR -A inject -v"
+	"fi_msg_inject -A inj_complete -v"
+	"fi_msg_inject -NR -A inj_complete -v"
```

Comment:
Not sure what the 'R' is doing here after the N for the above 2 calls.

Reply:
I missed the R here. All checks are passing now.